app/services/aspace_client.rb

```diff
@@ -5,20 +5,29 @@
 
 # Client for making requests via the ArchivesSpace API
 class AspaceClient
-  attr_reader :url
+  attr_reader :base_url
 
-  def initialize(url: ENV.fetch('ASPACE_URL', nil),
+  # rubocop:disable Metrics/MethodLength
+  def initialize(url: Settings.aspace.url,
```

Comment:
Will this disrupt environments where `ASPACE_URL` is configured? https://github.com/search?q=repo%3Asul-dlss%2Fpuppet%20ASPACE_URL&type=code

Reply:
It won't disrupt any environments where `ASPACE_URL` is set because the value of `Settings.aspace.url` is set to `ENV['ASPACE_URL']` in `settings.yml`